astro/src/content/docs/apis/_import-users-request-body.mdx

```diff
@@ -27,266 +27,335 @@ You must provide either the **email** or the **username** field for each User. T
+</APIField>
 
-    Unless you are using `TextMessage` as your delivery type, ensure you are able to share the secret with the User before enabling Two Factor authentication. Beginning in version 1.17.0, if you do create a User with `TextMessage` set as the `twoFactorDelivery` type and you omit this value, the secret will be generated for you. The secret can be generated because it is not necessary to share the secret with the User for this delivery method.
+<APIField name="users[x].passwordChangeRequired" type="Boolean" optional defaults="false">
+  Indicates that the User's password needs to be changed during the next login attempt.
+</APIField>
 
-    When using `None` as the `twoFactorDelivery` this value will be required.
+<APIField name="users[x].passwordLastUpdateInstant" type="Long" optional defaults="now">
+  Indicates that last [instant](/docs/reference/data-types#instants) the password was changed.
 
-    <RemovedSince since="1.26.0" />
-  </APIField>
-  <APIField name="users[x].username" type="String" optional>
-    The username of the User.
-  </APIField>
-  <APIField name="users[x].usernameStatus" type="String" optional defaults="ACTIVE">
-    <ModerationStatusRequest />
-  </APIField>
+  If `encryptionScheme` is omitted this value will be ignored and set to now.
 
-  <APIField name="users[x].verified" type="Boolean" optional defaults="false">
-    Set this value to `true` in order to mark the user as having a verified email address.
-  </APIField>
+  Note that if you have enabled password expiration or plan to do so in the future and you set this value to `0` or some other arbitrary time in the past that is greater than the password expiration value in days, these users will be required to change their password. It is recommended that you omit this value and allow the default value to be set unless you know the last time the user changed their password.
+</APIField>
 
-  <APIField name="users[x].verifiedInstant" type="Long" optional defaults="now">
-    The [instant](/docs/reference/data-types#instants) this user's email address was verified. This value will be ignored if <InlineField>verified</InlineField> is `false`.
-  </APIField>
+<APIField name="users[x].phoneNumber" type="String" optional since="1.99.9">
+  The phone number of the User. The phone number is stored and returned in E.164 canonical format, however a phone number is considered unique regardless of the format. `303-555-1212` is considered equal to `+13035551212` so either version of this phone number can be used whenever providing it as input to an API.
 
-  <APIField name="validateDbConstraints" type="Boolean" optional defaults="false">
-    Set this value to `true` in order to perform additional validation of the request.
+  If phoneNumber is not provided, then <InlineField>email</InlineField> or <InlineField>phoneNumber</InlineField> will be required.
```

Comment:
I think this should be `username`. Why do we call out required identity fields here but not on `email` or `username`?

Perhaps this import request parameter should just say "The phone number of the User." and omit the remaining details like the other fields?
```suggestion
  If phoneNumber is not provided, then <InlineField>email</InlineField> or <InlineField>username</InlineField> will be required.
```